packages/ComponentLibrary/src/colors.ts

```diff
@@ -110,3 +111,7 @@ export const PROCESS_900 = '#78320C';
 
 // Draft color
 export const DRAFT_MAIN = '#E5E5E6';
+
+// Dynamic color
+export const DYNAMIC_MAIN = '#004ACA';
```

Comment:
I've created a new task and asigned to you to (finally) import the color palette.